presto-raptor/src/main/java/com/facebook/presto/raptor/systemtables/ShardMetadataRecordCursor.java

```diff
@@ -65,6 +68,10 @@
     private static final String TABLE_NAME = "table_name";
     private static final String MIN_TIMESTAMP = "min_timestamp";
     private static final String MAX_TIMESTAMP = "max_timestamp";
+    private static final String MIN_DATE = "min_date";
+    private static final String MAX_DATE = "max_date";
+
+    private static final String NULL_COLUMN = "null";
```

Comment:
I'd prefer to inline this, as it makes it more clear what's happening. It's not actually a column -- rather, it's selecting a SQL null.

Reply:
Done.